dts/bindings/sensor/espressif,esp32-temp.yaml

```diff
@@ -8,17 +8,16 @@ compatible: "espressif,esp32-temp"
 include: sensor-device.yaml
 
 properties:
-  range:
+  range-min:
     type: int
     description: |
-      The temperature sensor is available on the ESP32-S2, ESP32-S3, ESP32-C3, ESP32-C6.
-      Note that it is unavailable on the ESP32 due to missing offset calibration.
-      Temperature range is defined by the temperature offset which is used during
-      calculation of the output temperature from the measured value.
-    default: 2
-    enum:
-    - 0    # measure range: 50°C ~ 125°C, error < 3°C
```

Comment:
I think it is worth to keep in the sensor description the available ranges (min/max) for the temperature sensor. The default values of -10C and 80C are the most accurate and makes sense to be the default.